web/src/i18n/locales/en.yaml

```diff
@@ -48,12 +48,12 @@
             zip: ZIP
     overview: 
       title: "Program Overview"
-      helpText: "Provide a high level summary of your state's program, history, and future vision."
+      helpText: "Provide a high level summary of your state's program, history, and future vision. This initial overview is intended to be brief, there will be opportunities to explain in more detail in the activity details section."
       reminder: "Indicate the federal fiscal years for which this APD is requesting funding."
       yearsCovered: "What federal fiscal year(s) does this APD cover?"
       labels: 
         desc: "Program Introduction"
-        helpText: "Describe the scope of what you intend to do. It is not meant to replicate the executive summary section. You will have an opportunity to explain in more detail later in this application."
+        helpText: "Describe the scope of what you intend to do. This summary will be used in other parts of this application. You will have an opportunity to explain in more detail in the activity details section."
```

Comment:
For some reason the helptext here is sitting above the Program Introduction section, as compared to HIE and MMIS where it sits below. Not sure if this is fixable here, or in another part of the app. Thoughts?

Reply:
The `Program Introduction` part doesn't have the same layout as the other two for some reason.  That'll need to be a dev fix, but super easy.  I'll add it on this branch so it'll go in at once.

Reply:
"This initial overview is intended to be brief, **there will be opportunities to explain in more detail in the activity details section.**"

This reads like a run-on sentence(?) how about (catching tonal changes where we can):

"Provide a high level summary of the state's program, history, and future vision. Keep it brief. Specific activities for the program will be explained in the program activities section."